### PR TITLE
no code change

### PR DESCRIPTION
**Pull request by meichstedt**
just for testing :)

### CODE REVIEW

File: foo [hunk 1]
```diff
@@ -0,0 +1 @@
+bar
```
Comment (gkleiman) [inline foo]: This should be baz.

Comment (meichstedt) [inline foo]: I tried baz, but it didn't fit well into our architecture.

**Comment by meichstedt** [comment]
Currently, both the velocity _and_ TeamCity builds have failed here. As the commit message states, there was no code change.

**Comment by aquamatthias** [comment]
I like the baz architecture!

**Pull request closed without merging**